src/Factory/StripeTransferFactory.php

```diff
@@ -79,24 +79,26 @@ public function createFromOrder(MiraklOrder $order): StripeTransfer
     {
         if (is_a($order, MiraklServiceOrder::class)) {
             $type = StripeTransfer::TRANSFER_SERVICE_ORDER;
+            $pendingDebits = $this->miraklClient->listServicePendingDebitsByOrderIds([$order->getId()]);
```

Comment:
You could pass the order pending debit to createFromOrder here: https://github.com/stripe/stripe-mirakl-connector/pull/113/files#diff-7c7f8bc083b40f4e555e014f9b311bf776a0a3799cf0588d295f2b3fb9b7278eL67
You used this approach for updateFromOrder so the object is already available if I'm reading the code right.

Reply:
Good catch, thanks!